examples/hero/hero_shooter_2023.cc

```diff
@@ -126,18 +126,19 @@ void RM_RTOS_Default_Task(const void* args) {
   }
 
   while (true) {
-    // Dead
-//    while (Dead || GimbalDead) osDelay(100);
-
     // whether change the force motor position
-    ForceWeakDetect.input(dbus->keyboard.bit.F || dbus->swr == remote::UP);
-    ForceStrongDetect.input(dbus->keyboard.bit.C || dbus->swr == remote::UP);
+    ForceWeakDetect.input(dbus->keyboard.bit.F || dbus->wheel.wheel > remote::WheelDigitalValue);
+    ForceStrongDetect.input(dbus->keyboard.bit.C
+                            || (dbus->wheel.wheel == remote::WheelDigitalValue && dbus->previous_wheel_value == remote::WheelDigitalValue));
+
     // just execute force transform once
     if (ForceWeakDetect.posEdge() && !ForceStrongDetect.posEdge() && !force_week) {
```

Comment:
force_week -> force_weak
Also fix lines 138, 143, 155, 157, definition in line 116